src/openklant/components/klantinteracties/api/tests/test_digitaal_adres.py

```diff
@@ -90,6 +92,54 @@ def test_create_digitaal_adres(self):
             self.assertEqual(data["adres"], "foobar@example.com")
             self.assertEqual(data["omschrijving"], "omschrijving")
 
+    def test_create_digitaal_adres_email_validation(self):
```

Comment:
Could you also add a test which verifies that whenever `SoortPartij` is not `email` no email validation is done?

Reply:
This is already tested as part of `test_update_digitaal_adres` which uses `SoortDigitaalAdres.telefoonnummer`, is that enough or should I add a separate test case for it?